datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -28,6 +44,37 @@
 
     private static GraphQLEngine _engine;
 
+    public static final DatasetType DATASET_TYPE = new DatasetType(GmsClientFactory.getDatasetsClient());
+    public static final CorpUserType CORP_USER_TYPE = new CorpUserType(GmsClientFactory.getCorpUsersClient());
+    public static final DataPlatformType DATA_PLATFORM_TYPE = new DataPlatformType(GmsClientFactory.getDataPlatformsClient());
+    public static final DownstreamLineageType DOWNSTREAM_LINEAGE_TYPE = new DownstreamLineageType(GmsClientFactory.getLineagesClient());
+
+    /**
+     * Configures the graph objects that can be fetched primary key.
+     */
+    public static final List<LoadableType<?>> LOADABLE_TYPES = ImmutableList.of(
+            DATASET_TYPE,
```

Comment:
when would this not be all types? trying to think of a counter example

Reply:
This is all types. This is where we define what "all types" is!

Reply:
Are you suggesting a renaming?

Reply:
That's fine- this works.